### DISCUSSION TITLE
Hero11 Black needs to read the QR code twice

### DISCUSSION THREAD

**Discussion by Fedess1986** [gopro/labs#950] [General]
Hello Gopro Team!

I 'm facing this issue with my Hero11 Black (Labs firmware v2.30.70)

When I set this everything goes fine: 
![image](https://github.com/gopro/labs/assets/134018336/422c3012-e79f-4023-b402-5738586c816d)

But If I set the previous QR and then I read this QR:
![image](https://github.com/gopro/labs/assets/134018336/8c1525f3-e835-4c3a-a952-5a401d5a2eb4)

the 4k Linear + Horizon Lock is not enabed. 4k Wide is enabled. When I scan the same QR for the 2nd time, it enables 4k Linear + Lock

**Comment by dnewman-gpsw** [comment]
Yes it does, but I moved this to discussions as this is by design. What commands are the minimum to cause the failure? You would have discovered the error.  This command are performed in order, mV, then r4, etc.  You have an issue switching between fW and fH.  Try ```mVr4Xp30fWe0``` then ```mVr4p60fHe1```  see the same issue? 
 
Now try ```mVr4Xp30e0!NfW``` then ```mVr4p60e1!NfH``` 

What happening is you turning EIS off with e0, if you look in the menu EIS is required to support Linear + Horizon Lock (it is ghosted out.)  So when you run the second command you asked for fH before e1, so it will correctly fail. 

The parts I have to fix: the e1 switch is taking too long, so I had to add !N after.  I will add this to the web tool, and change the order.

**Comment by Fedess1986** [comment]
Thank you for your clarification David! 
I tested it and worked fine. Also the !N command to workaround the delay issue did fine 😁
